pkg/rbac/sharedData.go

```diff
@@ -140,39 +141,46 @@ func (cache *Cache) GetPropertyTypes(ctx context.Context, refresh bool) (map[str
 	}
 }
 
-func (cache *Cache) PopulateSharedCache(ctx context.Context) error {
+func (cache *Cache) PopulateSharedCache(ctx context.Context) {
+	defer metric.SlowLog("PopulateSharedCache", 200*time.Millisecond)()
```

Comment:
should this be 300*time.Millisecond?

Reply:
In this case it is overriding the default (300ms) for this specific function to be lower (200ms).
With this we can set a more accurate SlowLog based on what each function is doing.